packages/eslint-plugin/docs/rules/README.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Overview
+sidebar_label: Overview
+pagination_next: null
+pagination_prev: null
+slug: /
+---
+
```

Comment:
What do you think about inserting a blurb here about the two main rule config lists? Vaguely:

```suggestion

`@typescript-eslint/eslint-plugin` comes with two rulesets you can extend from to pull in the recommended starting rules:

* `'plugin:@typescript-eslint/recommended'`: unopinionated rules for code correctness you can drop in without additional configuration. See [Linting](../../linting/README.md) for more details.
* `'plugin:@typescript-eslint/recommended-requiring-type-checking'` additional recommended rules that require type information. See [Linting](../../linting/TYPED_LINTING.md) for more details.

```

Reply:
for now, i added it as kinda placeholder, i do agree that we should add more description and maybe reorganize this file